packages/ndla-ui/src/Resource/BlockResource.tsx

```diff
@@ -89,33 +92,40 @@ const BlockInfoWrapper = styled.div`
   gap: ${spacing.xxsmall};
 `;
 
-const ImageWrapper = styled.div`
+const ImageWrapper = styled.div<ImageContentType>`
   display: flex;
   width: 100%;
+  justify-content: center;
   overflow: hidden;
   align-items: center;
-  div {
-    min-width: 100%;
-  }
+  background-color: ${({ contentType }) => handleColorType(contentType)};
+  height: 137px;
   img {
     min-width: 100%;
   }
 `;
-
+interface ImageContentType {
+  contentType: string;
+}
 interface BlockImageProps {
   image: ResourceImageProps;
   loading?: boolean;
+  contentType: string;
 }
 
-const BlockImage = ({ image, loading }: BlockImageProps) => {
-  if (loading) {
-    return (
-      <ContentLoader height={'100%'} width={'100%'} viewBox={null} preserveAspectRatio="none">
-        <rect x="0" y="0" rx="3" ry="3" width="100%" height="100%" />
-      </ContentLoader>
-    );
+const BlockImage = ({ image, loading, contentType }: BlockImageProps) => {
+  if (image.src === '') {
+    return <ContentTypeBadge type={contentType} size="large" />;
```

Comment:
ContentTypeBadge har støtte for background som prop (boolean). Om vi bruker dette fjerner vi kanskje behov for å mappe farger i handleColorType? Tror dette skal skje automatisk

Reply:
Background går dessverre bare på bakgrunnen inne i ikon-elementet
<img width="92" alt="image" src="https://user-images.githubusercontent.com/25527932/191688191-f0294c55-102a-4f68-9606-62b00ea052f3.png">

Reply:
Aha, ser det nå 👍